emotiontech_pc.xml.fdm_material

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" version="1.3">
+    <metadata>
+        <name>
+            <brand>eMotionTech</brand>
+            <material>PC</material>
+            <color>Generic</color>
+			<label>PC</label>
+        </name>
+        <GUID>1b29dbfd-0193-427d-a126-a4f8069fbad1</GUID>
+        <version>14</version>
+        <color_code>#DBFAF9</color_code>
+        <description> </description>
+        <adhesion_info>We recommend using a brim for 0.4mm nozzles, but it is not necessary for bigger nozzles</adhesion_info>
+    </metadata>
+    <properties>
+        <density>1.2</density>
+        <diameter>1.75</diameter>
+        <weight>2300</weight>
+    </properties>
+    <settings>
+        <setting key="print temperature">280</setting>
+        <setting key="heated bed temperature">120</setting>
+        <setting key="standby temperature">150</setting>
+        <setting key="retraction amount">2</setting>
+        <setting key="build volume temperature">60</setting>
+        <machine>
+            <machine_identifier manufacturer="eMotionTech" product="Strateo3D"/>
+            <hotend id="Standard 0.4" >
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">270</setting>
+                <setting key="retraction amount">3</setting>
+            </hotend>
+            <hotend id="Standard 0.6" >
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">270</setting>
+                <setting key="retraction amount">2</setting>
+            </hotend>
+            <hotend id="Standard 0.8">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">280</setting>
+                <setting key="retraction amount">2</setting>
+            </hotend>
+            <!-- Commented out until eMotionTech adds their hotend profiles to Cura.
+            <hotend id="Standard 1.0">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">225</setting>
+                <setting key="retraction amount">1.5</setting>
+            </hotend>
+            <hotend id="Standard 1.2">
+                <setting key="hardware compatible">yes</setting>
+                <setting key="print temperature">245</setting>
+                <setting key="retraction amount">1.9</setting>
+            </hotend>
+            -->
```

Comment:
How come this is commented out? And why are the hotends here not named "Experimental" like in the rest of the materials?

Reply:
I created a new PR based on this #156 , I add the "Experimental".

In addition, I answered why it was commented out.